Kajabi/sage-lib
# Sortable Item Custom - added new rails subcomponent

## Description

This PR is intended to set a pattern when creating a custom panel. The aim is to reduce some of the rigidity with layouts being tightly coupled in the component. In some cases, the design pattern of the component already exists within our [Grid Templates](http://sage-design-system.kajabi.com/pages/layout/grid_templates). Interested in feedback on this approach to add flexibility while still confined to our sage principles

### Screenshots
![Screen Shot 2021-04-01 at 3 46 03 PM](https://user-images.githubusercontent.com/1241836/113354743-3d730600-9305-11eb-8ad4-494230df277c.png)

## Test notes

1. Visit the Sortable Item Rails page: http://localhost:4000/pages/object/sortable
2. Check the `Using a custom grid template` section and verify

### Estimated impact

- [ ] #400 (**LOW**) This is a new component that does not currently exist within the app.



## Related
